ext/storage_sources/s3_store/README.md

```diff
@@ -0,0 +1,77 @@
+# WiredTiger's S3 Store Extension
+
+## Building and running 
+
+This is a guide to build WiredTiger with the S3 extension enabled.
```

Comment:
I suggest moving your description of the cmake flags here. It will give a natural introduction to the following subsections.

For example:

There are two cmake flags associated with the S3 extension: `ENABLE_S3` and `IMPORT_S3_SDK`.

- `ENABLE_S3=1` is required to build the S3 extension
- `IMPORT_S3_SDK={external,package}`
  - **external** tells the compiler to search for an existing system installation of the S3 SDK
  - **package** tells the compiler to download and install the S3 SDK as part of the build

If the `IMPORT_S3_SDK` flag is not specified, the compiler will assume a system installation of the SDK.

Reply:
Wouldn't it be easier to have a flag named `DOWNLOAD_S3_SDK` that is either `0` or `1`? Using "import" to mean "download" is not typical usage, and the options "external" and "package" are confusing as well.

Reply:
I think it is common in the CMake files to use import, external, and package.